### ISSUE TITLE
[rcp] Compile warning, implicit conversion

### ISSUE THREAD

**Issue by ejurgensen** [owntone/owntone-server#1471]
@whatdoineed2do please take a look at this:

```
outputs/rcp.c:1216:14: warning: implicit conversion from enumeration type 'enum output_device_state' to different enumeration type 'enum rcp_state' [-Wenum-conversion]
s->state = OUTPUT_STATE_STOPPED;
         ~ ^~~~~~~~~~~~~~~~~~~~
```

**Comment by whatdoineed2do** [comment]
This was fixed in the PR #1461 that’s ready to merge if no other comments from your side

**Comment by ejurgensen** [comment]
Suppose you meant 1459, which is now merged
